### PR TITLE
solution 0.1

### PR DESCRIPTION
**Pull request by PyCore23**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1 +1,22 @@
-# write your code here
+from .cafe import Cafe
+from .errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list[dict], cafe: "Cafe") -> None:
```
Comment (vasylhnatiuk) [inline app/main.py:5]:
```suggestion
def go_to_cafe(friends: list[dict], cafe: Cafe) -> None:
```

**Review by vasylhnatiuk — CHANGES_REQUESTED** [review]

File: app/errors.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+class VaccineError(BaseException):
+    ...
```
Comment (EdAlekseiev) [inline app/errors.py:2]: Use `pass` instead of `...`

File: app/cafe.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import datetime
+from .errors import (NotVaccinatedError, OutdatedVaccineError, NotWearingMaskError)
```
Comment (EdAlekseiev) [inline app/cafe.py:2]: use absolute path instead of relative

**Review by EdAlekseiev — CHANGES_REQUESTED** [review]

**Review by vsmutok — APPROVED** [review]
Good Job!

**Pull request closed without merging**